e2e/cypress/integration/opla/create_user.spec.js

```diff
@@ -43,7 +43,9 @@ context('Actions', () => {
     cy.get('#signin-dialog-username').type(username)
     cy.get('#signin-dialog-password').type(password)
     cy.contains('Sign in').click()
+    cy.contains('.userbox_name', username)
```

Comment:
why that ?

Reply:
To check that Sign in worked. We check that the username is displayed in the top bar